server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreBaseIntegTestCase.java

```diff
@@ -360,4 +362,21 @@ protected void prepareCluster(int numClusterManagerNodes, int numDataOnlyNodes,
             ensureGreen(index);
         }
     }
+
+    protected void prepareCluster(
+        int numClusterManagerNodes,
+        int numDataOnlyNodes,
+        String indices,
+        int replicaCount,
+        int shardCount,
+        Settings settings
+    ) {
+        internalCluster().startClusterManagerOnlyNodes(numClusterManagerNodes, settings);
+        internalCluster().startDataOnlyNodes(numDataOnlyNodes, settings);
+        for (String index : indices.split(",")) {
+            createIndex(index, remoteStoreIndexSettings(replicaCount, shardCount));
+            ensureYellowAndNoInitializingShards(index);
+            ensureGreen(index);
```

Comment:
For my understanding, cant we just use `ensureGreen`? Why do we need to check for `yellow` first?

Reply:
Yes we can just use ensureGreen as well.

Reply:
I see `ensureYellow` check is still there. Not sure if this is intended?

Reply:
Removed it.